### PR TITLE
Ported to work with latest version of i18next v3.4.1, examples and unit tests are working, using TypeScript, and output to UMD format using rollup.js

### PR DESCRIPTION
**Pull request by anwalkers**
This isn't ready for prime time but is gets my project where it needs to be. I can put a few more cycles into it updating docs etc. Probably belongs in a v3 branch.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place your settings in this file to overwrite default and user settings.
```
Comment (archer96) [inline .vscode/settings.json]: This file (and directory) should have been added to `.gitignore`.

File: gulpfile.js [hunk 2]
```diff
@@ -4,97 +4,99 @@ var gulp = require('gulp');
...
+var tsconfig = require("./tsconfig.json");
+
+
+var getToday = function () {
 
 	var today = new Date();
-	var dd    = today.getDate();
-	var mm    = today.getMonth() + 1; //January is 0!
-	var yyyy  = today.getFullYear();
+	var dd = today.getDate();
```
Comment (archer96) [inline gulpfile.js]: Why remove the spaces? :)

**Comment by archer96** [comment]
Hello,

thank you very much for this pull request. :+1: 
I won't merge this into master (for now) but would like to have a new branch (as you mentioned). You have to close this PR and create a new (see: http://stackoverflow.com/questions/24159036/how-to-modify-a-pull-request-on-github-to-change-target-branch-to-merge-into/24159161 ).

Regards,
Andre

File: gulpfile.js (hunk 2 above)
Comment (anwalkers) [inline gulpfile.js]: VS code formatting. I will restore.

**Comment by anwalkers** [comment]
Moving this to v2 branch

**Pull request closed without merging**